bittide-instances/README.md

```diff
@@ -64,3 +64,14 @@ Example:
 ```
 cabal run -- bittide-instances:shake scatterUnitWb:netlist
 ```
+
+## Hardware
+Prerequisites to running shake:
+- Source VivadoEnterprise
```

Comment:
We should probably move this to a separate section at the very start of the README. E.g..,

```
# Prerequisites

 * Vivado 2222.2
```